src/components/atoms/Button.module.css

```diff
@@ -66,6 +67,23 @@
   background: var(--brand-white);
 }
 
+.outline {
+  background: linear-gradient(
+    to right,
+    var(--color-primary) 50%,
+    var(--brand-white) 50%
+  );
+  background-size: 202% 100%;
+  background-position: bottom right;
+  border: 2px solid var(--color-primary);
```

Comment:
question(non-blocking): Is there some global value for border thickness? It would be great to be consistent here and allow for an easy update.

Reply:
There is only a variable for the border radius, not for the border size. Probably because all the other button styles are color filled and so they don't have a border. It would be a variable used mainly in this component and maybe in the navbar, but we can create it anyway, no problem

Reply:
Ideally, these values would follow global values. A designer taught me that good design also tries to minimize variation and only adds it if there is no alternative or better said if it improves the overall design.

It's a detail, but I'd go for the variable here and replace it for the navbar and this case.